### ISSUE TITLE
[UX] Re-word the 'Congratulations' installation message

### ISSUE THREAD

**Issue by ghost** [backdrop/backdrop-issues#4162]
<img width="952" alt="Screen Shot 2019-10-28 at 7 31 15 am" src="https://user-images.githubusercontent.com/2423362/67641254-03097a80-f955-11e9-808a-56ae13ae123d.png">

Does the

> Congratulations, you installed Backdrop CMS!

message you get after installation seem to imply that Backdrop is hard to install and therefore, if you do it successfully, you deserve congratulating...?

Maybe it should be simply:

> Welcome to your Backdrop site!

(or something along those lines)

Apparently I'm not the first person to think this. @Bojhan made a similar comment back in 2016: https://github.com/backdrop/backdrop-issues/issues/1908#issuecomment-221297323

**Comment by jenlampton** [comment]
I would love to change the welcome message :) What does WordPress say when it's installed?

**Comment by domaingood** [comment]
I would like to suggest Add Welcome page with some segregation to setup and how to manage help after install.So new user can easily understand where and how to do a simple demonstration with videos document

**Comment by BWPanda** [comment]
@domaingood Your suggestions I think are covered by the Dashboard page, which has all those sorts of links and features. But maybe we should link to the Dashboard from the welcome message...?

**Comment by klonos** [comment]
> What does WordPress say when it's installed?

They say "WordPress has been installed. Thank you, and enjoy".

We automatically log the user in, and redirect them to the home page; whereas WP displays a dedicated screen/step to confirm that everything was installed successfully:

<img width="768" alt="Screen Shot 2019-10-28 at 7 41 09 am" src="https://user-images.githubusercontent.com/2423362/67641515-a4450080-f956-11e9-9fd6-d0907b2a9d06.png">

They then require an additional click on a "Log in" button 👎 ...unless there is a very good reason for doing that, I think we are doing a much better job UX-wise in this step.

How about we say something like this?: "Thank you for installing Backdrop! We've built it with ❤️ for you to enjoy."

> But maybe we should link to the Dashboard from the welcome message...?

Yes 👍 ...we could also have a link there to request feedback; something like "Please let us know what we've nailed, and how we could improve further."

**Comment by olafgrabienski** [comment]
To me, the message "Congratulations, you installed Backdrop CMS!" never implied that Backdrop is hard to install but I see that "Congratulations" may imply it to others.

I'd suggest to stay with something simple instead, maybe not "Welcome to your Backdrop site!" (it's already part of the hero) but maybe just "You installed Backdrop CMS!" or "Backdrop has been installed." A short phrase like this would better fit into the flow of the page:

- Welcome hero block
- Message
- Your first post

> "Thank you for installing Backdrop! We've built it with ❤️ for you to enjoy."

Sounds a bit wordy to me, and the focus on "us" building Backdrop doesn't go so well with "Your first post", in my opinion.

**Comment by stpaultim** [comment]
First of all, I don't have a problem with "congratulations." It seems appropriate for me. I think it's appropriate to feel a sense of accomplishment after successfully spinning up a new site, which for beginners is not always as easy as we would like. 

Having said that, I'm also good with a thank you message. 

"Thank you for installing Backdrop CMS! Please visit your dashboard for tips on getting started." (include link)

**Comment by klonos** [comment]
Noting that if we are to place a message and link encouraging people to visit the Dashboard, we should first be checking if the module is enabled.

**Comment by stpaultim** [comment]
> Noting that if we are to place a message and link encouraging people to visit the Dashboard, we should first be checking if the module is enabled.

Well, this message only happens at the end of the install proces. If the user uses the standard install profile, it will always be enabled. But, if they are using a different install profile, it might not be.

**Comment by klonos** [comment]
> Well, this message only happens at the end of the install process. If the user uses the standard install profile, it will always be enabled. But, if they are using a different install profile, it might not be.

Yes, this message seems to be in https://github.com/backdrop/backdrop/blob/1.x/core/includes/install.core.inc#L1714 which picks the "Backdrop CMS" bit from https://github.com/backdrop/backdrop/blob/1.x/core/includes/install.inc#L211

> The distribution name defined in the profile's .info file. Defaults to "Backdrop CMS" if none is explicitly provided by the installation profile.

...so yeah, `install_finished()` has the possibility to be called by any installation profile - not just the Standard profile. This means that the Dashboard module may not have been enabled, hence why I suggested that we should add the check 😉

**Comment by pumpkinsk3** [comment]
I think a short message is always better because in my opinion I lose interest if the message is too long (too many words to read). If this is just an indicator of the success of installation, I think we can use something like "Great, you successfully installed Backdrop CMS!"

**Comment by Nimisha94** [comment]
Just tested this. This is what it looks like for me. LGTM!

<img width="1438" alt="Screen Shot 2020-10-01 at 3 36 31 PM" src="https://user-images.githubusercontent.com/13430900/94855278-42113d00-03fc-11eb-884a-6835e8351634.png">

**Comment by stpaultim** [comment]
This is a pretty simple text change, so I went ahead and marked it RTBC.

**Comment by stpaultim** [comment]
NOTE: We worked on this issue together as part of vGHC Open Source Day. We decided to break this issue into two parts.

#1) A simple text change
#2) Create a follow up issue about linking to documentation. 

Could someone from OSD, please create a followup issue. In the follow-up issue, link to this issue and describe the part of the task that we did not finish here.

**Comment by BWPanda** [comment]
Awesome, I like it! Nice and simple, doesn't imply difficulty, while still indicating installation was a success.

**Comment by BWPanda** [comment]
I'm ready to merge this. Someone want to confirm this can go into the next bugfix release by adding the milestone?

**Comment by jenlampton** [comment]
yes please :)

**Comment by BWPanda** [comment]
Thank you @pumpkinsk3 for the PR! Thanks also to @Nimisha94 and @stpaultim for the review. I've merged https://github.com/backdrop/backdrop/pull/3340 into 1.x and 1.17.x.
